src/NLog/Targets/ColoredConsoleTarget.cs

```diff
@@ -299,14 +299,14 @@ private void WriteToOutput(LogEventInfo logEvent, string message)
                 }
                 catch (IndexOutOfRangeException ex)
                 {
-                    //this is a bug and therefor stopping logging. For docs, see PauseLogging property
+                    // This is a bug and will therefore stopp the logging. For docs, see the PauseLogging property.
```

Comment:
isn't it "stop"?

Reply:
Ah, my bad... of course! Thanks, will fix.